src/pages/FormBuilder/FormBuilder.module.css

```
@@ -367,51 +270,75 @@
+
 .sectionTitleActionsDeleteDisabled {
   opacity: 1;
 }
 
-.sectionActionBtn {
-  background: none;
-  border: none;
-  border-left: 1px solid var(--gray-200);
-  color: var(--gray-500);
-  border-radius: 0;
+:global(button:disabled) {
```

Stylelint (17.9.0)

[error] Unknown pseudo-class selector ":global" (selector-pseudo-class-no-unknown)

(selector-pseudo-class-no-unknown)